### PR TITLE
Update to API 4.2.0

### PR DESCRIPTION
**Pull request by Uukrull**
I've been using this change since the API update on a daily basis and didn't notice any regressions.

### CODE REVIEW

File: src/client.cpp [hunk 1]
```diff
@@ -688,4 +688,5 @@ void PauseStream(bool _UNUSED(bPaused)) {}
 bool SeekTime(int, bool, double*) { return false; }
 void SetSpeed(int) {};
 bool IsTimeshifting(void) { return false; }
+bool IsRealTimeStream(void) { return true; }
```
Comment (ksooo) [inline src/client.cpp]: IsTimeshifting alsways returning false in conjunction with IsRealTimeStream always returning true makes sence, but I doubt that IsTimeshifting is implemented correctly (yeah I know that this was not changed by this PR) as  implementation of the other timeshifting functions implies that this addon actually supports timeshifting => example: https://github.com/kodi-pvr/pvr.dvbviewer/pull/28/files#diff-34f48ad91ac6c202ac60b0348ae90e30R580

So, if this addon actually supports timeshifting, imo IsTimeshifting and IsRealTimeStream both need to be fixed to take actual timeshifting state into account.

@FernetMenta any opinions on that?

Comment (manuelm) [inline src/client.cpp]: As explained in the dvbviewer subforum (you probably haven't read this, but @Uukrull might have), I'll adapt the PVR in the next days. `IsTimeshifting` returning false is definitely wrong.

Btw, when is feature freeze for Krypton?

Comment (Jalle19) [inline src/client.cpp]: In half a year maybe.

Comment (Uukrull) [inline src/client.cpp]: > As explained in the dvbviewer subforum (you probably haven't read this, but @Uukrull might have), I'll adapt the PVR in the next days.

I didn't read that. If you're going to update the PVR with the API changes, this PR can be closed.

**Comment by manuelm** [comment]
Closed in favor of #30

**Pull request closed without merging**